freeipa/src/main/java/com/sequenceiq/freeipa/controller/UserV1Controller.java

```diff
@@ -76,16 +92,17 @@ public SetPasswordResponse setPassword(SetPasswordRequest request) {
     }
 
     @Override
-    public CreateUsersResponse createUsers(CreateUsersRequest request) {
+    public SetPasswordResponse getSetPasswordStatus(String operationId) {
+        if (operationId == null) {
+            throw new IllegalArgumentException("operationId must be provided");
+        }
         try {
-            String accountId = crnService.getCurrentAccountId();
-            userService.createUsers(accountId, request);
+            LOGGER.debug("getPasswordStatus() requested for operation {}", operationId);
+            return passwordService.getSetPasswordStatus(operationId);
         } catch (Exception e) {
-            LOGGER.error("Failed to create users", e);
-            throw new RuntimeException(e);
+            LOGGER.error("getPasswordStatus caught exception. rethrowing", e);
```

Comment:
this is not really necessary this way as we have exceptionmappers for controllers which log the errors

Reply:
hmm.. I thought so too, but I hadn't seen all the exceptions in the logs. I'll check again and look at the exception mappers

Reply:
removed